src/Momento.Sdk.Incubating/Responses/CacheDictionaryGetFieldResponse.cs

```diff
@@ -107,12 +91,29 @@ public string Message
             get => $"{_error.MessageWrapper}: {_error.Message}";
         }
 
-        public byte[] FieldByteArray
+        public byte[]? FieldByteArray
         {
-            get => field.ToByteArray();
+            get
+            {
+                if (field != null)
+                {
+                    return field.ToByteArray();
+                }
+                return null;
+            }
         }
 
-        public string FieldString { get => field.ToStringUtf8(); }
+        public string? FieldString
+        {
+            get
+            {
+                if (field != null)
+                {
+                    return field.ToStringUtf8();
+                }
+                return null;
+            }
+        }
```

Comment:
We can simplify these with .NET 's null coalescing operator:

```csharp
public byte[]? FieldByteArray
{
    get => field?.ToByteArray();
}
```

Reply:
Ohh!